Notes_and_Examples/5_CurrentFlow/5_currentDirection.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+        <link href="/c4x/DavidsonNext/Ann101/asset/vectordraw.css" rel="stylesheet" type="text/css" />
+    </head>
+
+    <body>
+        <div id="vectordraw" />
+        <script type="text/javascript">
+          // Background image, vectors, and points are defined below.
+          var vectordraw_settings = {
+              width: 550,
+              height: 450,
+              bounding_box_size: 8,
+              background: {
+                  src: '/c4x/DavidsonNext/Ann101/asset/imageCircuitDirection.png',
+                  width: 13
+              },
+              vectors: [
+                    {name: 'I<sub>1</sub>', description: 'Current through R<sub>1</sub>', coords: [[4, 4], [7, 7]], render: false},
+                    {name: 'I<sub>2</sub>', description: 'Current through R<sub>2</sub>', coords: [[3, 3], [6, 6]], render: false},
+              ],
+              points: [{name: 'cm1', coords: [-0.2,-0.25]},
+                       {name: 'cm2', coords: [-0.2,-5.3]}],
+              expected_result: {
+                 'I<sub>1</sub>': {angle: 0, angle_tolerance: 5, tail: [-0.2,-0.25], tail_tolerance: 1.0},
```

Comment:
@dseaton cool that it works, but as far as I know it's an undocumented feature - use with caution, we haven't explicitly built support for html markup in vector labels/names/etc., so warranty does not cover it :) Anyway, I believe It should just work most of the time, but if anything goes wrong - please let us know and we'll see what we can do.